foundation/foundation-mda/src/test/resources/specifications/data-records-generation.feature

```diff
@@ -181,7 +181,7 @@ Feature: Generating pipeline resources
       | data-flow | data-delivery-pyspark |
 
   @pipeline-generation @module-generation
-  Scenario Outline: Habushu pipelines integrate test report generation by default
+  Scenario Outline: Spark pipelines integrate test report generation by default
```

Comment:
A: "Spark" can imply java.  I think we want this to say "Spark and pyspark" or "(Py)Spark" or something like that.